src/utilities/params.spec.ts

```diff
@@ -13,7 +13,6 @@ describe('getParamValue', () => {
   test('returns for correct value for Number', () => {
     expect(getParamValue('1', Number)).toBe(1)
     expect(getParamValue('1.5', Number)).toBe(1.5)
-    expect(() => getParamValue('', Number)).toThrowError()
```

Comment:
You removed this because an empty string will never be passed to the getter yeah? Could we add a failing example like `foo` (anything that returns `NaN`.

Reply:
correct. I think through the course of this PR we agreed that empty strings will never match a param unless it's optional, and in that case it would be replaced with `undefined`. But yeah, it's probably worth having a case where these tests throw an error 👍

Reply:
I actually updated all the tests that used an empty string to use `foo` instead, you cool with that?

Reply:
Yeah that makes sense to me 👍